crates/bevy_pbr/src/lib.rs

```diff
@@ -136,6 +139,12 @@ impl Plugin for PbrPlugin {
             "render/pbr_ambient.wgsl",
             Shader::from_wgsl
         );
+        load_internal_asset!(
+            app,
+            PBR_FRAGMENT_HANDLE,
+            "render/pbr_fragment.wgsl",
```

Comment:
Does the name `pbr_fragment` mean something like “use this in the fragment shader for pbr-based materials”? Do we need a separate file for this? Would the contents not make sense under the existing `pbr.wgsl` file?

Reply:
i chose to separate them mainly to make the reusable `pbr_fragment` api clearer. i do agree the actual `pbr.wgsl` is pretty basic now, but it feels nicer to me this way, and arguably makes for an easier "first step" into the shader code for people browsing the source. 

including it also potentially blocks allowing imported modules to specify shader entry points in future (though naga_oil doesn't support imported entry points anyway right now, i think it could do in future without too much effort).

i'd rather keep this as is unless there's a concensus against it.